pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1798,4 +1798,13 @@ self: super: {
     passthru.updateScript = ../../../maintainers/scripts/haskell/update-cabal2nix-latest.sh;
   };
 
+  # Install man page and generate shell completions
+  pinboard-notes-backup = overrideCabal
+    (generateOptparseApplicativeCompletion "pnbackup" super.pinboard-notes-backup)
+    (drv: {
+      postInstall = ''
+        install -D man/pnbackup.1 $out/share/man/man1/pnbackup.1
+      '' + (drv.postInstall or "");
+    });
+
```

Comment:
So, I think technically this belongs into configuration-nix.nix since it’s independent of the specific versions we are using.

Reply:
I had asked peti on friday if we still need the distinction between common and nix, but I think his explanation was not bad.

Reply:
I mean there is no technical distinction, it is purely semantic (they are always both applied). Since this is inconsistently handled we may want to get rid of this at some point, but I guess, I'm gonna move this one.